mimesis/data/gr/personal.json

```diff
@@ -0,0 +1,5619 @@
+	"Our Brand Is Crisis",
+	"Our Family Wedding",
+	"Our Idiot Brother",
+	"Our Kind of Traitor",
+	"Out Cold",
+	"Out of Africa",
+	"Out of Inferno",
+	"Out of Sight",
+	"Out of Time",
+	"Out of the Blue",
+	"Out of the Blue",
+	"Out of the Dark",
+	"Out of the Furnace",
+	"Outbreak",
+	"Outlander",
+	"Outside Bet",
+	"Outside Providence",
+	"Over Her Dead Body",
+	"Over the Hedge",
+	"Over the Hill to the Poorhouse",
+	"Owning Mahowny",
+	"Oz the Great and Powerful",
+	"Oz the Great and Powerful",
+	"P.S. I Love You",
+	"PCU",
+	"Paa",
+	"Pacific Rim",
+	"Paddington",
+	"Pain & Gain",
+	"Paint Your Wagon",
+	"Pale Rider",
+	"Palo Alto",
+	"Pan",
+	"Pan",
+	"Pan",
+	"Pan's Labyrinth",
+	"Pandaemonium",
+	"Pandora's Box",
+	"Pandorum",
+	"Panic Room",
+	"Paparazzi",
+	"Paper Towns",
+	"ParaNorman",
+	"Paranormal Activity",
+	"Paranormal Activity 2",
+	"Paranormal Activity 3",
+	"Paranormal Activity 4",
+	"Paranormal Activity: The Marked Ones",
+	"Parental Guidance",
+	"Paris, je t'aime",
+	"Parker",
+	"Partition",
+	"Party Monster",
+	"Passchendaele",
+	"Pat Garrett & Billy the Kid",
+	"Patch Adams",
+	"Pathology",
+	"Patriot Games",
+	"Patton",
+	"Paul",
+	"Paul Blart: Mall Cop",
+	"Paul Blart: Mall Cop 2",
+	"Pay It Forward",
+	"Payback",
+	"Paycheck",
+	"Peace, Propaganda & the Promised Land",
+	"Peaceful Warrior",
+	"Pearl Harbor",
+	"Peeples",
+	"Peggy Sue Got Married",
+	"Penguins of Madagascar",
+	"Penitentiary",
+	"People I Know",
+	"Perception",
+	"Percy Jackson & the Olympians: The Lightning Thief",
+	"Percy Jackson: Sea of Monsters",
+	"Perfect Cowboy",
+	"Perfume: The Story of a Murderer",
+	"Perrier's Bounty",
+	"Persepolis",
+	"Pet Sematary",
+	"Pete's Dragon",
+	"Phantasm II",
+	"Phat Girlz",
+	"Phenomenon",
+	"Philadelphia",
+	"Philomena",
+	"Phone Booth",
+	"Pi",
+	"Pieces of April",
+	"Pierrot le Fou",
+	"Pineapple Express",
+	"Pink Flamingos",
+	"Pink Narcissus",
+	"Pink Ribbons, Inc.",
+	"Pinocchio",
+	"Piranha 3D",
+	"Pirate Radio",
+	"Pirates of the Caribbean: At World's End",
+	"Pirates of the Caribbean: Dead Man's Chest",
+	"Pirates of the Caribbean: On Stranger Tides",
+	"Pirates of the Caribbean: The Curse of the Black Pearl",
+	"Pitch Black",
+	"Pitch Perfect",
+	"Pitch Perfect 2",
+	"Pixels",
+	"Planet 51",
+	"Planet of the Apes",
+	"Planet of the Apes",
+	"Plastic",
+	"Platoon",
+	"Play It to the Bone",
+	"Playing for Keeps",
+	"Please Give",
+	"Plush",
+	"Pocahontas",
+	"Pocketful of Miracles",
+	"Poetic Justice",
+	"Point Blank",
+	"Point Break",
+	"Point Break",
+	"Pokémon 3: The Movie",
+	"Police Academy",
+	"Police Academy: Mission to Moscow",
+	"Polisse",
+	"Pollock",
+	"Poltergeist",
+	"Poltergeist",
+	"Poltergeist III",
+	"Pompeii",
+	"Pontypool",
+	"Ponyo",
+	"Pooh's Heffalump Movie",
+	"Poolhall Junkies",
+	"Pootie Tang",
+	"Porky's",
+	"Poseidon",
+	"Post Grad",
+	"Poultrygeist: Night of the Chicken Dead",
+	"Pound of Flesh",
+	"Practical Magic",
+	"Preacher",
+	"Precious",
+	"Precious",
+	"Predator",
```

Comment:
Why all names in English? Do you have not localization of the names of movies?

Reply:
These movies do have Greek translations but pretty much everyone uses their English names since the translations are really bad. If it's absolutely i will change some of them (translating all of them would take me ages)

Reply:
No no . Use English names, if so taken in Greece. You are a Greek and you know better.